__tests__/api/app.test.js

```diff
@@ -4,6 +4,7 @@ const seedTestData = require('../../db/data/test-data');
 const seed = require('../../db/seeds/seed');
 const db = require('../../db/connection');
 const endpoints = require('../../endpoints.json');
+const { expect } = require('@jest/globals');
```

Comment:
Looks like a sneaky auto import made its way into your tests. Be sure to remove this.